bin/generate_test_list.py

```diff
@@ -38,6 +38,7 @@ def get_paths(level=15):
     Generates a set of paths for testfiles searching.
 
     Example:
```

Comment:
Okay.  I hadn't been changing those because I wanted to keep a light touch, but that won't be hard.